plugins/modules/network/routeros/routeros_api.py

```diff
@@ -0,0 +1,494 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Nikolay Dachev <nikolay@dachev.info>
+# GNU General Public License v3.0+ https://www.gnu.org/licenses/gpl-3.0.txt
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0.1',
```

Comment:
Is the metadata_version must match version_added ?

Reply:
No, actually the whole `ANSIBLE_METADATA` block needs to be removed. It is no longer necessary, and in fact will result in linting errors eventually.